sdk/keyvault/azure-security-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/AzureKeyVaultConfiguration.java

```diff
@@ -7,6 +7,6 @@
     //TODO: Eventually remove these hardcoded strings with https://github.com/Azure/azure-sdk-for-java/issues/3141
     static final String SDK_NAME = "Azure-Keyvault";
     // {x-version-update-start;com.azure:azure-security-keyvault-secrets;current}
-    static final String SDK_VERSION = "4.1.0-beta.1";
+    static final String SDK_VERSION = "4.0.1";
```

Comment:
@g2vinay was this missed during the KeyVault release?

Reply:
I don't know keyvault details, but I did look into this, and this is what is in client_version.txt and is unmodified by my changes. So the version script just picked this up and it looks like maybe it was somehow skipped earlier. I can revert it if that's preferable.

Reply:
yeah, it looks like it was missed.
The release hasn't happened yet.
@hemanttanwar , lets restart the release pipeline when this merges.

Reply:
Thanks for having this in your PR @rickle-msft. @g2vinay, the update script updates versions that are hard coded in source files as well, the PR that updated azure-security-keyvault-secrets should have had this file included. In the future, when you run the update script, any source files with version changes also need to be in the PR, not just the POM and README files.